packages/perseus/src/widgets/number-line/number-line.test.ts

```diff
@@ -20,32 +20,137 @@ describe("number-line widget", () => {
         );
     });
 
-    it("should snapshot", () => {
-        // Arrange
-        const apiOptions: APIOptions = {
-            isMobile: false,
-        };
-
-        // Act
-        const {container} = renderQuestion(question1, apiOptions);
-
-        // Assert
-        expect(container).toMatchSnapshot("first render");
-    });
-
-    it("should snapshot on mobile", () => {
-        // Arrange
-        jest.useRealTimers();
-
-        const apiOptions: APIOptions = {
-            isMobile: true,
-        };
-
-        // Act
-        const {container} = renderQuestion(question1, apiOptions);
-
-        // Assert
-        expect(container).toMatchSnapshot("first mobile render");
+    describe("snapshots", () => {
+        let originalOptions;
+        let apiOptions: APIOptions;
+
+        beforeAll(() => {
+            originalOptions = snapshots.widgets["number-line 1"].options;
+            apiOptions = {
+                isMobile: false,
+            };
+        });
+
+        afterAll(() => {
+            snapshots.widgets["number-line 1"].options = originalOptions;
```

Comment:
It's a little unusual to mutate test data like this, and makes the tests harder to understand. If someone sees `snapshots` being used in a test, they might reasonably (but erroneously, in this case) conclude that it's the value being imported from `./number-line.testdata`.

I'd recommend copying the data with modifications instead:

```ts
const question = {
  ...snapshots,
  widgets: {
    ...snapshots.widgets,
    "number-line 1": {
      ...snapshots.widgets["number-line 1"],
      options: widgetOptions,
    },
  },
};

const {container} = renderQuestion(question, apiOptions);
```

You could create a `createNumberLineQuestionWithOptions()` helper function to de-duplicate the resulting code.